protobuf/api/service.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+import "protobuf/definition/service.proto";
+
+package api;
+
+service ServiceX {
```

Comment:
I put X at the end for now otherwise names conflicts for _api.Service_

Reply:
hum that's annoying. Ultimately the service api (with emitEvent, listenTask) will be removed in favor of the `execution/stream` api and a `core/emit` api. So I'm ok to keep it like that and rename it when we will be at this step.
Maybe if you can just add a TOFIX with a comment to make sure we don't forget about it

Reply:
done